tests/integration/ha/continuous_writes.py

```diff
@@ -168,16 +169,17 @@ def _client(self):
         )
 
     @staticmethod
-    async def _run(event: Event, data_queue: Queue, starting_number: int) -> None:  # noqa: C901
+    async def _run(
+        event: Event, data_queue: Queue, starting_number: int, ops_test
+    ) -> None:  # noqa: C901
         """Continuous writing."""
-        initial_data = data_queue.get(True)
```

Comment:
**todo someday**: not in this PR, but if you pass ops_test to ContinuousWrites process, the queue is no longer needed. It could be kept for future use, but right now, removing the initial_data is only triggering this block:
```pyhton
if not data_queue.empty():  # currently evaluates to false as we don't make updates
       data_queue.get(False)
       client.close()
       client = _client()
```